### PR TITLE
Hardcoded Bearer tokens in an Authorization header

### PR DESCRIPTION
**Pull request by aegilops**
Authorization: Bearer <some token>

Still needs: test cases, testing at CLI, testing in UI

### CODE REVIEW

File: generic/patterns.yml [hunk 1]
```diff
@@ -70,3 +69,22 @@ patterns:
         start_offset: 131
         end_offset: 167
 
+  - name: Bearer Tokens
```
Comment (aegilops) [inline generic/patterns.yml:72]: This is where the actual changes are! The rest is merging with `main`

**Comment by aegilops** [comment]
I've added tests now, and run offline against test data with 0 FPs

File: generic/patterns.yml [hunk 2]
```diff
@@ -70,3 +69,29 @@ patterns:
...
+    regex:
+      version: 0.1
+      pattern: |
+        [a-zA-Z0-9_.=/+:-]+
+      start: |
+        ['"\s][Aa]uthorization: Bearer[ ]+
+      end: |
+        \z|[\s'"]
+      additional_not_match:
+        - ^(?:letmein|Oracle|SuperSecretString|foo|ababbdbbebbbebdbbe5538003023|XYZ_INVALID_ACCESTOKEN_XYZ|QQ==|Shizuku|mF_9.B5f-4.1JqM|h480djs93hd8|SlAV32hkKG)$
```
Comment (GeekMasher) [inline generic/patterns.yml:83]: Are most of these just False Positives where people have hard coded these values? Is there a significance to some of the values are the end?

Comment (aegilops) [reply generic/patterns.yml:83]: They're things from docs and samples mostly. The final ones are from OAuth README files

**Review by GeekMasher — APPROVED** [review]
Amazing, nice work!

**Pull request merged**